bindings/kepler.gl-jupyter/notebooks/DataFrame.ipynb

```diff
@@ -53,45 +54,45 @@
    ],
    "source": [
     "import keplergl\n",
-    "w1 = keplergl.KeplerGl()\n",
+    "w1 = keplergl.KeplerGl(height=600, data={'data_1': df})\n",
     "w1"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": 18,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Map saved to keplergl_map_time_test_1.html!\n"
+      "Map saved to keplergl_map.html!\n"
      ]
     }
    ],
    "source": [
-    "w1.save_to_html(file_name='keplergl_map_time_test_1.html')"
+    "w1.save_to_html(file_name='keplergl_map.html')"
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 2",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "python2"
+   "name": "python3"
```

Comment:
@ajduberstein @kylebarron FYI

Reply:
I think this just means that the notebook was run using a Python 3 session instead of Python 2, and shouldn't be an issue.

Reply:
Yes, that makes sense. Was mainly pointing it out from an alignment perspective, do we do the same in pydeck etc? 

Not sure if it matters, but always keen to look for patterns and best practices, and also just wanted to make sure pydeck folks have visibility in kepler's Juypter related PRs.

Reply:
This is metadata that is automatically added by Jupyter, so I think it should be the same in every notebook, as long as it was run with Python 3.